### PR TITLE
[cfl] Allow longer to repro issues in Python

### LINKED ISSUE #9222: clusterfuzz-lite: unable to reproduce instant crash for Python fuzzer
This fuzzer https://github.com/ossf/fuzz-introspector/blob/main/src/test/fuzz/test_fuzz_report_generation.py finds a bug almost instantly when run. However, our clusterfuzz-lite integration didn't catch the bug as a reproducible bug, because it's determined not reproducible:

```
2022-12-15T22:21:07.7681386Z 2022-12-15 22:21:07,767 - root - INFO - Deleting fuzz target: test_fuzz_cfg_load.
2022-12-15T22:21:07.7686641Z 2022-12-15 22:21:07,768 - root - INFO - Done deleting.
2022-12-15T22:21:07.7691746Z 2022-12-15 22:21:07,768 - root - INFO - Fuzzer test_fuzz_cfg_load finished running without reportable crashes.
2022-12-15T22:21:07.7698642Z 2022-12-15 22:21:07,769 - root - INFO - Running fuzzer: test_fuzz_report_generation.
2022-12-15T22:21:07.7704455Z 2022-12-15 22:21:07,770 - root - INFO - Downloading corpus for test_fuzz_report_generation to /github/workspace/cifuzz-corpus/test_fuzz_report_generation.
2022-12-15T22:21:09.2788102Z 2022-12-15 22:21:09,278 - root - WARNING - Could not find artifact: cifuzz-corpus-test_fuzz_report_generation.
2022-12-15T22:21:09.2789685Z 2022-12-15 22:21:09,278 - root - WARNING - Could not download artifact: cifuzz-corpus-test_fuzz_report_generation.
2022-12-15T22:21:09.2795603Z 2022-12-15 22:21:09,279 - root - INFO - Done downloading corpus. Contains 0 elements.
2022-12-15T22:21:09.2800914Z 2022-12-15 22:21:09,279 - root - INFO - Starting fuzzing
2022-12-15T22:24:39.2897882Z 2022-12-15 22:24:39,286 - root - INFO - Fuzzer: test_fuzz_report_generation. Detected bug.
2022-12-15T22:24:39.2898173Z 
2022-12-15T22:24:39.2898906Z 2022-12-15 22:24:39,287 - root - INFO - Trying to reproduce crash using: /tmp/tmp3hl_1tt2/tmpqvsmvugt.
2022-12-15T22:25:09.2965686Z 2022-12-15 22:25:09,295 - root - ERROR - Reproducing timed out..
2022-12-15T22:25:09.2972083Z 2022-12-15 22:25:09,296 - root - INFO - Crash is not reproducible.
```

I think problem is that the fuzzer has a long startup time, and this start up time is more than the 30 seconds set for reproducer time (set here I think https://github.com/google/oss-fuzz/commit/2c6a7dde490d6b6041cce73401c4313976e8a979). The start up time is due to Atheris instrumenting the target when the python script is initialised and in our case I think it will take about a minute. The specific line in the above fuzzer that causes the long instrumentation time is: https://github.com/ossf/fuzz-introspector/blob/0146c4ad97d1fe3f0d4a841cac860fdcce5edc85/src/test/fuzz/test_fuzz_report_generation.py#L80

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Fixes #9222

### CODE REVIEW

**Review by Alan32Liu — APPROVED** [review]
LGTM, but why do we need more time for python?

**Comment by jonathanmetzman** [comment]
> LGTM, but why do we need more time for python?

Python fuzzers spend an inordinate amount of time starting up. See the attached issue.

File: infra/cifuzz/fuzz_target.py [hunk 1]
```diff
@@ -284,13 +285,18 @@ def is_reproducible(self, testcase, target_path, reproduce_args):
     with clusterfuzz.environment.Environment(config_utils.DEFAULT_ENGINE,
                                              self.config.sanitizer,
                                              target_path):
+
+      if self.config.language != 'python':
```
Comment (oliverchang) [inline infra/cifuzz/fuzz_target.py:289]: Nit: this double negative in the else part is a bit confusing.

Also, perhaps we can make this a dict instead, and add more code comments to provide context to future readers:

```python
PER_LANGUAGE_TIMEOUTS = {
  'python': 30 * 4  # Python takes a bit longer on startup.
}

reproduce_time_seconds = PER_LANGUAGE_TIMEOUTS.get(self.config.language, DEFAULT_REPRODUCE_TIME_SECONDS)
````

**Review by oliverchang — APPROVED** [review]
Thanks @Alan32Liu !

**Comment by Alan32Liu** [comment]
@jonathanmetzman @oliverchang Shall we merge?

**Comment by oliverchang** [comment]
LGTM.

**Pull request merged**